java/ql/lib/change-notes/2023-11-03-jdk21-support.md

```diff
@@ -0,0 +1,5 @@
+---
+category: minorAnalysis
+---
+* Switch cases using patterns and both cases and instanceof expressions using record patterns are now supported. The new class `RecordPatternExpr` is introduced to represent patterns, `PatternCase` and `CaseNullDefault` to represent new kinds of case statement, `InstanceOfExpr` gains `getPattern` to replace `getLocalVariableDeclExpr`.
```

Comment:
`Switch cases using patterns and both cases ...` is a little confusing to read. I think splitting this into 2 bullet points might be better.